src/Appium.Net/Appium/AppiumDriver.cs

```diff
@@ -534,6 +535,91 @@ public string StopRecordingScreen(IScreenRecordingOptions options)
 
         #endregion Recording Screen
 
+        #region Compare Images
+
+        /// <summary>
+        /// Performs images matching by features. Read
+        /// https://docs.opencv.org/3.0-beta/doc/py_tutorials/py_feature2d/py_matcher/py_matcher.html
+        /// for more details on this topic.
+        /// </summary>
+        /// <param name="base64Image1">base64-encoded representation of the first image</param>
+        /// <param name="base64Image2">base64-encoded representation of the second image</param>
+        /// <param name="options">comparison options</param>
+        /// <returns>The matching result. The configuration of fields in the result depends on comparison options</returns>
+        public FeaturesMatchingResult MatchImageFeatures(string base64Image1, string base64Image2, FeaturesMatchingOptions options = null)
```

Comment:
Addressing the "nullable parameter" from #354:
To my knowledge, there is no equivalent to "Nullable" parameters in C#. 
However, I've used [optional parameters](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments), which are probably the closest equivalent.